Externals/wxWidgets3/include/wx/msw/app.h

```diff
@@ -29,7 +29,7 @@ class WXDLLIMPEXP_CORE wxApp : public wxAppBase
     virtual ~wxApp();
 
     // override base class (pure) virtuals
-    virtual bool Initialize(int& argc, wxChar **argv);
+    virtual bool Initialize(int& _argc, wxChar **_argv);
```

Comment:
Er, won't the diffs be easily viewable in git history? Also, the base revision (should) be kept here: https://github.com/dolphin-emu/dolphin/blob/master/Externals/wxWidgets3/build_wx.sh#L3